File: PizzaManagerEmployee/PizzaManagerEmployee/model/ShoppingCartItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PizzaManagerEmployee.model
{
    public class ShoppingCartItem
    {
        public string _id { get; set;}
        public string productId { get; set;}
        public string orderId { get; set; }
        public int quantity { get; set; }

        public Product product { get; set; }
    }
}

File: PizzaClient/app/src/main/java/com/sdproject/szabi/pizzaclient/datamodel/ShoppingCartItem.java
package com.sdproject.szabi.pizzaclient.datamodel;

import com.sdproject.szabi.pizzaclient.datamodel.products.Product;

/**
 * Created by Szabi on 5/25/2017.
 */

public class ShoppingCartItem {

    public Product product;
    public int quantity;
    public String orderId;
    public String productId;

    public ShoppingCartItem(){

    }

    public ShoppingCartItem(Product product, int quantity){
        productId = product._id;
        this.product = product;
        this.quantity = quantity;
    }
}
